### ISSUE TITLE
Logs shown double after a little while

### ISSUE THREAD

**Issue by xriter** [amir20/dozzle#2398]
**Describe the bug**
I can't exactly pinpoint when or why this is happening, but if I have Dozzle open in a tab and some logs are being shown, and a little later I come back to that tab, some of those logs are printed again, and again, and again... So then it looks like as if that log message happened multiple times, but this isn't the case. When I refresh the page, that particular log message is only shown once (how it's supposed to be). So, somewhere along the track Dozzle lost track of which logs have already been printed to the console...

**Expected behavior**
Log messages shouldn't be shown multiple times if they only happend once.

**Screenshots**
<img width="1512" alt="dozzle" src="https://github.com/amir20/dozzle/assets/70454441/df5f3c78-c973-4556-8884-52ee0351faf8">

**Desktop:**
- OS: Mac OS
- Docker version 24.0.5
- Browser & version safari
- Version 5.0.3

**Comment by amir20** [comment]
Thanks for the screenshots. I have actually seen this happen very rarely. Some questions for you: 

1. Are you using https? 
2. Does this only happen in Safari? Next time, could you you `Show javascript web console` and see if you see errors? 
3. When you switch between containers, are the old logs still there? Without refreshing the whole application. 

I have seen this myself in Chrome. It was related to HTTPS in IPv6. I searched every where and the error was `ERR_NETWORK_CHANGED` which made absolutely no sense to me.  Disabling IPv6 fixed it. But it would be nice if I didn't have to do that. 

Let me know what you find.

**Comment by amir20** [comment]
Hmm it might be something different because for me the logs are still there between containers. They just go away after going to home. 

I'll have to think how else one could end up in this situation. Let me know if you see if any errors. I also don't test in Safari much so I should do that more.

**Comment by amir20** [comment]
Hmm Failed to load resource with a timeout sound confusing. I think I'll need to add more logging. I really like to get to the bottom of this because it's so confusing. I think something wonky happens EventStreams and I don't close it properly.

**Comment by amir20** [comment]
So far in my testing, Dozzle has been running Safari for 15 minutes and no repeats. How long does it usually take?

**Comment by amir20** [comment]
Hi @xriter,

Can you try `amir20/dozzle:pr-2406`? I have added more logging #2406. 

I also clear the logs now when an error is received. It might actually fix your problem. 

Let me know!

**Comment by amir20** [comment]
Any update? It seems to be working OK for me. 

So I have noticed that sometimes right _AFTER_ clicking on a container, the connection throws an exception. Which causes old logs to be there and possibly get the page in a bad state. 

I fixed this in #2406 by clearing all logs after any exception. This is not ideal because then the logs go away for a while.  But it did make my problems go away. 

I cannot figure out why the EventSource socket throws an exception, but clearing it seems the best next option.
